### PR TITLE
Ruby in Twenty Minutes

### PR DESCRIPTION
**Pull request by ErvinaDe**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -470,7 +470,7 @@
 
 * [Learn how to program: Ruby](https://www.learnhowtoprogram.com/ruby) - Epicodus Inc.
 * [RubyMonk - Interactive Ruby tutorials](https://rubymonk.com)
-
+* [Ruby in Twenty Minutes](https://www.ruby-lang.org/en/documentation/quickstart/)
```
Comment (SethiPandi) [inline free-courses-en.md:474]: There should be 2 line breaks above a new section.
Could you add the line break back?

**Review by SethiPandi — COMMENTED** [review]
You may want to read the contribution guidelines (CONTRIBUTING.md) for guidance on formatting!

File: free-courses-en.md (hunk 1 above)
Comment (ErvinaDe) [reply free-courses-en.md:474]: Yes, sorry, my bad.

**Comment by SethiPandi** [comment]
Closing this as the resources are already in the repository.

**Pull request closed without merging**